index.html

```diff
@@ -5,21 +5,39 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link href="style.css" rel="stylesheet" />
+    <link href="desktop_style.css" rel="stylesheet" />
+    <script src="pop-up-menu.js" defer></script>
     <title>My Portfolio</title>
   </head>
   <body>
-    <header>
+
+    <!-- header section -->
+    <header class="mobile-version desktop-version">
       <nav class="hamburger-menu">
-        <ul class="bars">
+        <ul class="menu-desktop-version">
+          <li class="element0">
+            <a class="hello-item" href="#hello">Hello</a>
+          </li>
+          <li class="element1">
+            <a class="portfolio-item" href="#portfolio">Portfolio</a>
+          </li>
+          <li class="element2">
+            <a class="about-item" href="#about">About</a>
+          </li>
+          <li class="element3">
+            <a class="contact-item" href="#contact">Contact</a>
+          </li>
+        </ul>
```

Comment:
- [ ] There are some issues here, kindly consider fixing them.
  - The mobile menu is not responsive so consider making it responsive. you mobile menu section has fixed width and padding on both sides. kindly check in that in given screenshot and consider fixing it.
  - You can remove link underlines by applying `text-decoration: none`.
  - Kindly consider placing the close button in the right place as it is placed in the same place of the hamburger menu in the figma design.
  your design
![r](https://github.com/norbiboy1/Portfolio/assets/96683843/c355b31d-d4c8-44bf-8f9c-9518d6e44031)
![t](https://github.com/norbiboy1/Portfolio/assets/96683843/78bd8c2c-14af-4ff8-aa17-a82fbba2084d)

figma design
![qw](https://github.com/norbiboy1/Portfolio/assets/96683843/0c7e9a4d-d2c8-4b3d-9d61-e1b932145311)